mock/py/mockbuild/config.py

```diff
@@ -728,6 +730,52 @@ def load_defaults(uidManager, version, pkg_python_dir):
     return setup_default_config_opts(gid, version, pkg_python_dir)
 
 
+def print_description(config_path, config_filename, uidManager, version, pkg_python_dir):
+    (basename, dirname, basename_without_ext) = parse_config_filename(config_filename)
+    config_opts = load_config(config_path, config_filename, uidManager, version, pkg_python_dir)
```

Comment:
Can you please catch exceptions from this method?  I can see this on my box:
mockbuild.exception.ConfigError: Could not find included config file: /etc/mock/epel-8-aarch64.cfg

That's because of:
```
$ rpm -qf /etc/mock/epel-8-aarch64-rpmfusion_free.cfg 
mock-rpmfusion-free-36.0-1.fc35.noarch
```